src/homework5/bin_gd.py

```diff
@@ -0,0 +1,24 @@
+"""Greatest divisor with the power of two.
```

Comment:
task for binary operators

Reply:
Sure. My solution doesn't use binary shifts, but i does definitely use binary logic. Imagine binary 2: it is '10', 4 is '100', 16 is '10000', 48 is '110000'. Power of two is already in the binary representation of the given number. So the power of two is equal to the number of right zeros in the binary representation. That what my algorithm does.

Reply:
I agree and understand your solution. but task was for binary operations.